api/client/samples/analysis_kits/stocks_to_use_price_model.py

```diff
@@ -0,0 +1,287 @@
+"""Sample API script for modeling prices based on the stocks-to-use ratio
+
+This script creates a basic CME price valuation model for corn and soybeans
+using the US stocks-to-use ratio. 
+
+The stocks-to-use ratio is calculated each month as reported in the USDA's PS&D
+database. The price variable is defined as the monthly average 'new crop' 
+(December futures for corn and November futures for soybeans) price for the 
+relevant report month. For example, the corn-June scatterplot will compare the US
+stocks-to-use ratio from all June reports relative to the corresponding monthly 
+average December corn futures price.
+
+The output of this script will be a PDF file for each modeled crop, saved in
+the current working directory. Each PDF will have 13 pages in total, 
+including a scatterplot for each report month as well as a 
+scatterplot summarizing the entire sample.
+
+"""
+
+import os
+import calendar
+
+import pandas as pd
+
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_pdf import PdfPages
+
+from sklearn import linear_model
+
+from api.client.gro_client import GroClient
+
+API_HOST = 'api.gro-intelligence.com'
+TOKEN    = os.environ['GROAPI_TOKEN']
+client   = GroClient(API_HOST, TOKEN)
+
+ref = {'corn':{'id':274, 'futures_contract_month':12},
```

Comment:
rather than hard-coding the item ids, better to use 

client.search_for_entity('items', 'corn')
client.search_for_entity('items', 'soybeans')